src/client/pages/signIn/SignIn.jsx

```diff
@@ -22,10 +25,12 @@ const SignIn = () => {
             body: JSON.stringify(loginDetails)
         })
         const data = await res.json();
-        //check data for error
-        //if error, return false
-        //else localStorage.setItem('auth', data.data) and return true
-        return data;
+        if (data.error) {
+            return false;
+        }
+        console.log(data)
```

Comment:
Remove printing statement please :)

Reply:
+1 (although I'm not going to let that stop me approving) - but _don't_ spam the console!